spec/generators/rspec/install/install_generator_spec.rb

```diff
@@ -27,7 +27,7 @@ def require_rspec_rails
     match(/^require 'rspec\/rails'$/m)
   end
 
-  def use_transactional_fixtures
+  def have_transactional_fixtures
```

Comment:
this feels like unnecessary churn to me, I don't think any of those names are particularly stronger than what we have right now.